ingestion-beam/src/main/java/com/mozilla/telemetry/util/KeyStore.java

```diff
@@ -0,0 +1,104 @@
+package com.mozilla.telemetry.util;
+
+import com.google.api.pathtemplate.ValidationException;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.io.Resources;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.security.PrivateKey;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.commons.io.IOUtils;
+import org.everit.json.schema.Schema;
+import org.everit.json.schema.loader.SchemaLoader;
+import org.jose4j.jwk.PublicJsonWebKey;
+import org.jose4j.lang.JoseException;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.json.JSONTokener;
```

Comment:
@jklukas Do you have insight into the following error?

```
[ERROR] /root/project/ingestion-beam/src/main/java/com/mozilla/telemetry/util/KeyStore.java:18:1: Illegal import - org.json.JSONObject. [IllegalImport]
```

Is this because `org.json` is not actually actually imported within the beam project? Do you think it's okay to pull in that dependency here?

Reply:
I believe this is coming from checkstyle. We specifically prohibit importing `org.json` so that we don't spread out JSON manipulation logic between two different formats. We should be doing all JSON manipulation with Jackson, and org.json should only come into play specifically for doing schema validation.

See https://github.com/mozilla/gcp-ingestion/blob/master/checkstyle/suppressions.xml#L15-L17 and https://github.com/mozilla/gcp-ingestion/blob/master/checkstyle/checks.xml#L68-L70

Reply:
I'm using this to load a schema for a configuration file, and then to read from that configuration file into the main data structure. This doesn't fit neatly within core, because of the `jose4j` dependency and the testing resources in the `pioneer` folder.

Reply:
Found the spot to put the checkstyle suppression -- it'd be necessary no matter what package it's part of.